core/spec.md

```diff
@@ -436,6 +440,117 @@ attributes. However they MUST adhere to the following rules:
   example, use of a model (or domain) specific prefix could be used to help
   avoid possible future conflicts.
 
+#### `xid`
+
+##### Referencing xRegistry Entities
+
+An `xid` is a URI according to
+[RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986) that
+uniquely identifies an entity within an xRegistry. Its purpose is to
+reference the entity and express a relation to it, even if it is not stored in
+the local registry. The `xid` can also be used in other places where it
+is helpful to use a URI, e.g. events or knowledge graphs.
+Each entity has exactly one `xid`. Therefore, comparing two `xid`
+values is possible to determine, if they identify the same or different
+entities. To achieve this, an xRegistry-specific URI scheme `xreg` is defined.
+Below, [ABNF](https://datatracker.ietf.org/doc/html/rfc5234)  notation is used
+to describe how `xids` are structured:
+
+`xid` = `"xreg://" authority "/" entity-path`
```

Comment:
based on the new "@" stuff, is this still correct?

Reply:
And, doesn't "authority" need to be defined either in the BNF or in the WHERE stuff below?

Reply:
Yes, for formal correctness, we will need more ABNF rules. When we have concluded on #181 I will add them.